docs/rules/max-lines.md

```diff
@@ -18,7 +18,7 @@ This rule has a number or object option:
 
 * `"skipComments": true` ignore lines containing just comments
 
-### code
+### max
```

Comment:
I'm not sure what was the meaning of "code", but "max" could be confusing since the examples present the number option, not the "max" property of the object option.

I think it would be best to just remove this heading.

Reply:
The heading is actual useful for addressing it: https://github.com/Shinigami92/eslint-define-config/blob/16ea76d92b1250283c26db5d84340a23fb39c411/src/rules/eslint/max-lines.d.ts#L14

But IMO it is the `number`-option or the `max` property
But `code` is nowhere in the document, therefor I think it was just a copy-paste issue of another rule or so

Reply:
Okay, makes sense to have the `max` heading here instead of no heading. `code` does look like a copy-paste error.